### ISSUE TITLE
Improve documentation for Style/TrailingCommaIn… cops

### ISSUE THREAD

**Issue by skalee** [rubocop-hq/rubocop#7536]
Three similar cops: `Style/TrailingCommaInArguments`, `Style/TrailingCommaInArrayLiteral`, and `Style/TrailingCommaInHashLiteral` share a common flaw. Their documentation does not show the difference between `comma` and `consistent_comma` settings. Neither code sample in documentation is marked as "good" for one setting and "bad" for another. A reader's impression is that these two settings are identical.

The source code of `TrailingComma` module reveals the difference between these two settings:

https://github.com/rubocop-hq/rubocop/blob/b21e8f222ec149a317a52b37b6b215a294c46486/lib/rubocop/cop/mixin/trailing_comma.rb#L65-L74

That is, if I understand it correctly:

* `consistent_comma` setting means that a trailing comma is required if and only if:
  * given list, hash etc. is multi-line, AND
  * neither item is on the same line of code as method name
* `comma` setting means that a trailing comma is required if and only if:
  * given list, hash etc. is multi-line, AND
  * every item is on its own line
  * surprisingly, it is allowed that one item is on the same line as method name, **which is possibly another bug** (I'm not opening a separate issue for that yet)

## Expected documentation

* Documentation on `Style/TrailingCommaInArguments` is expected to describe difference between `comma` and `consistent_comma` values for `EnforcedStyleForMultiline` attribute
* Documentation on `Style/TrailingCommaInArrayLiteral` is expected to describe difference between `comma` and `consistent_comma` values for `EnforcedStyleForMultiline` attribute
* Documentation on `Style/TrailingCommaInHashLiteral` is expected to describe difference between `comma` and `consistent_comma` values for `EnforcedStyleForMultiline` attribute

Possible values for `EnforcedStyleForMultiline` are not self-documenting at all. Few words about their meaning would be great. The mixin's source code may be a good source of inspiration.

At minimum, more examples should be added to show the difference between `comma`, and `consistent_comma` settings.

### Actual documentation for Style/TrailingCommaInArguments

The cop `Style/TrailingCommaInArguments` has option `EnforcedStyleForMultiline`, which is documented in https://rubocop.readthedocs.io/en/latest/cops_style/#styletrailingcommainhashliteral as follows:

> **EnforcedStyleForMultiline: consistent_comma**
> 
> ```ruby
> # bad
> method(1, 2,)
> 
> # good
> method(1, 2)
> 
> # good
> method(
>   1, 2,
>   3,
> )
> 
> # good
> method(
>   1,
>   2,
> )
> ```
> 
> **EnforcedStyleForMultiline: comma**
> 
> ```ruby
> # bad
> method(1, 2,)
> 
> # good
> method(1, 2)
> 
> # good
> method(
>   1,
>   2,
> )
> ```
> 
> **EnforcedStyleForMultiline: no_comma (default)**
> 
> (…)

### Actual documentation for Style/TrailingCommaInArrayLiteral

The cop `Style/TrailingCommaInArrayLiteral` has option `EnforcedStyleForMultiline`, which is documented in https://rubocop.readthedocs.io/en/latest/cops_style/#styletrailingcommainarrayliteral as follows:

> **EnforcedStyleForMultiline: consistent_comma**
> 
> ```ruby
> # bad
> a = [1, 2,]
> 
> # good
> a = [
>   1, 2,
>   3,
> ]
> 
> # good
> a = [
>   1,
>   2,
> ]
> ```
> 
> **EnforcedStyleForMultiline: comma**
> 
> ```ruby
> # bad
> a = [1, 2,]
> 
> # good
> a = [
>   1,
>   2,
> ]
> ```
> 
> **EnforcedStyleForMultiline: no_comma (default)**
> 
> (…)

### Actual documentation for Style/TrailingCommaInHashLiteral

The cop `Style/TrailingCommaInHashLiteral` has option `EnforcedStyleForMultiline`, which is documented in https://rubocop.readthedocs.io/en/latest/cops_style/#styletrailingcommainhashliteral as follows:

> **EnforcedStyleForMultiline: consistent_comma**
> 
> ```ruby
> # bad
> a = { foo: 1, bar: 2, }
> 
> # good
> a = {
>   foo: 1, bar: 2,
>   qux: 3,
> }
> 
> # good
> a = {
>   foo: 1,
>   bar: 2,
> }
> ```
> 
> **EnforcedStyleForMultiline: comma**
> 
> ```ruby
> # bad
> a = { foo: 1, bar: 2, }
> 
> # good
> a = {
>   foo: 1,
>   bar: 2,
> }
> ```
> 
> **EnforcedStyleForMultiline: no_comma (default)**
> 
> (…)

**Comment by anuja-joshi** [comment]
I will raise PR for this
